benchmarks/README.md

```diff
@@ -1,874 +1,26 @@
-# Benchmarking vLLM
+# Benchmarks
 
-This README guides you through running benchmark tests with the extensive
-datasets supported on vLLM. It’s a living document, updated as new features and datasets
-become available.
+This directory contains vLLM's benchmark scripts and utilities for performance testing and evaluation.
```

Comment:
```suggestion
This directory used to contain vLLM's benchmark scripts and utilities for performance testing and evaluation.

For detailed usage instructions, examples, and dataset information, see https://docs.vllm.ai/en/latest/contributing/benchmarks.html#benchmark-cli.

For full CLI reference see:

- https://docs.vllm.ai/en/latest/cli/bench/latency.html
- https://docs.vllm.ai/en/latest/cli/bench/serve.html
- https://docs.vllm.ai/en/latest/cli/bench/throughput.html
```

And then delete the rest of the content in this file. We want to direct all users to the docs.